chalicelib_cgap/app_utils.py

```diff
@@ -24,12 +24,7 @@ class AppUtils(AppUtils_from_core):
     prefix = FOURSIGHT_PREFIX
     FAVICON = 'https://cgap-dbmi.hms.harvard.edu/favicon.ico'
     host = HOST
-    package_name = 'chalicelib_cgap'
-
-    check_setup_file = AppUtils_from_core.locate_check_setup_file(os.path.dirname(__file__))
-    if not check_setup_file:
-        raise Exception("Unable to locate the check setup file!")
-    print(f"Using check setup file: {check_setup_file}")
+    AppUtils_from_core.package_name = 'chalicelib_cgap'
```

Comment:
It should be possible to import AppUtilsCore and not have to use this odd renamed class.

Reply:
Also, why isn't there a corresponding  line in `foursight`? I feel like these should be symmetric.